AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+huawei.com##.support-bootom-cookie
```

Comment:
Wait a second, do you need this rule for `huawei.com`? Only here it's necessary `consumer.huawei.com`.

Reply:
Ah yes. Not needed.